e2e/testcases/helm_sync_test.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2022 Google LLC
```

Comment:
did presubmit CA not catch this before?

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/37